.travis.yml

```diff
@@ -19,30 +19,79 @@ addons:
       - libgif-dev
       - google-chrome-stable
 
-install: # Install ChromeDriver (64bits; replace 64 with 32 for 32bits).
-  - wget -N https://chromedriver.storage.googleapis.com/78.0.3904.70/chromedriver_linux64.zip -P ~/
-  - unzip ~/chromedriver_linux64.zip -d ~/
-  - rm ~/chromedriver_linux64.zip
-  - sudo mv -f ~/chromedriver /usr/local/share/
-  - sudo chmod +x /usr/local/share/chromedriver
-  - sudo ln -s /usr/local/share/chromedriver /usr/bin/chromedriver
-
-before_script:
+install:
   - export PATH=./cmd/authelia-scripts/:/tmp:$PATH
-  - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
-  - nvm install v11 && nvm use v11 && npm i
   - source bootstrap.sh
 
 jobs:
   include:
-    - stage: test
+    - stage: build & test
+      before_script:
+        - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+        - nvm install v12 && nvm use v12
+        - go mod download
       script:
         - authelia-scripts --log-level debug ci
+
+    # Run all suites in a dedicated container
+    - &e2e-test
+      stage: end-to-end suite tests
+      env:
+        - SUITE_NAME=BypassAll
+      before_script:
+        # Install chrome driver
+        # TODO(c.michaud): this could be done in authelia-scripts instead for devs to not do the install themselves.
+        #                  or even provide a docker image with a selenium server.
+        - wget -N https://chromedriver.storage.googleapis.com/78.0.3904.70/chromedriver_linux64.zip -P ~/
+        - unzip ~/chromedriver_linux64.zip -d ~/
+        - rm ~/chromedriver_linux64.zip
+        - sudo mv -f ~/chromedriver /usr/local/share/
```

Comment:
Could this not just become `sudo mv -f ~/chromedriver /usr/local/bin/` and then perform the `chmod -x`?
That should be in the `$PATH` env var right, means you can get rid of the symbolic link too.